docs/advanced.md

```diff
@@ -3,7 +3,7 @@
 
 ## Custom MDX components
 
-mdx-deck includes default components for MDX, but to provide custom components to the [MDXProvider][], add a `components` object to the `theme`.
+mdx-deck includes default components for MDX, but to provide custom components to the [MDXProvider](https://github.com/mdx-js/mdx/blob/master/docs/getting-started/index.md#mdxprovider), add a `components` object to the `theme`.
```

Comment:
Thanks! Would you mind updating the reference link at the bottom of this docs instead of hard-coding the value here? It's also probably fine to link to https://mdxjs.com now that that's up

Reply:
Could you point to where I should make change. I'll be happy to make the change.

Reply:
Line 129

Reply:
Done. Thanks